src/conn_sock.c

```diff
@@ -71,11 +73,25 @@ struct remote_sock_s remote_notify_sock = {
 };
 
 /* External */
+
 char *setup_console_socket(void)
+{
+	return setup_socket(&console_socket_fd);
+}
+
+char *setup_seccomp_socket(void)
+{
+	return setup_socket(&seccomp_socket_fd);
+}
+
+static char *setup_socket(int *fd)
 {
 	struct sockaddr_un addr = {0};
-	_cleanup_free_ const char *tmpdir = g_get_tmp_dir();
-	_cleanup_free_ char *csname = g_build_filename(tmpdir, "conmon-term.XXXXXX", NULL);
+	const char *tmpdir = NULL;
+	_cleanup_free_ char *csname = NULL;
+
+	tmpdir = g_get_tmp_dir();
```

Comment:
why not initialize when we declare above? we do it in other places in conmon, and I think it's better to declare a variable right as it'll be used